docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,214 @@
+---
+id: dynamic-configuration
+title: Dynamic configuration reference
+description: Dynamic condifiguration key values can be set to override the default values in a Cluster configuration.
+sidebar_label: Dynamic configuration
+tags:
+  - reference
+---
+
+Temporal Cluster provides [dynamic configuration](/concepts/what-is-cluster-configuration#dynamic-configuration) keys that you can update and apply to a running Cluster without restarting your services.
+
+The dynamic configuration keys are set with default values when you create your Cluster configuration.
+You can override these values as you test your Cluster setup for optimal performance according to your workload requirements.
+
+For the complete list of dynamic configuration keys, see <https://github.com/temporalio/temporal/blob/master/common/dynamicconfig/constants.go>.
+Ensure that you check server releases notes for any changes to these keys and values.
+
+To check the default values set for a dynamic configuration key, check the following links:
+
+- [Frontend Service](https://github.com/temporalio/temporal/blob/5783e781504d8ffac59f9848b830868f3139b980/service/frontend/service.go#L176)
+- [History Service](https://github.com/temporalio/temporal/blob/5783e781504d8ffac59f9848b830868f3139b980/service/history/configs/config.go#L309)
+- [Matching Service](https://github.com/temporalio/temporal/blob/5783e781504d8ffac59f9848b830868f3139b980/service/matching/config.go#L125)
+- [Worker Service](https://github.com/temporalio/temporal/blob/5783e781504d8ffac59f9848b830868f3139b980/service/worker/service.go#L193)
+
+Note that setting dynamic configuration is optional.
+Change these values only if you need to override the default values to achieve better performance on your Temporal Cluster.
+Also, ensure that you test your changes before setting these in production.
+
+## Format
+
+To override the default dynamic configuration values, specify your custom values and constraints for the dynamic configuration keys that you want to change in a YAML configuration file.
+Use the following format when creating your dynamic configuration file.
```

Comment:
this whole section is kind of redundant with the following section now. maybe just get rid of it? or change it to a very high-level statement talking about yaml

Reply:
The idea was to give a quick reference handles for an overview on the format of a config file (#format) and details of defining the keys in your config file (#constraints).

Reply:
Does this read clearer? https://temporal-documentation-kcq4a95rs.preview.thundergun.io/references/dynamic-configuration#format